server/tests/test.js

```diff
@@ -0,0 +1,80 @@
+
+process.env.NODE_ENV = 'test';
+
+import anouncement from '../models/userAnouncement';
+
+import { should, use, request } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+should();
+
+
+use(chaiHttp);
+
+const announcementTestOne = {
+	"id":12,
+	"status": "pending",
+	"starts": "2000-12-20",
+	"ends": "20012-21-43",
+	"text": "addressser"
+}
+
+const changeAnounce = {
+  "status": 'pending'
+}
+
+  describe('anouncement test', () => {
+  
+      it('can create the anouncement', () => {
```

Comment:
All your test names should be descriptive. A good format is to start with `should create an announcement`